ci-operator/step-registry/ipi/conf/aws/local-zones/byo-vpc/ipi-conf-aws-local-zones-byo-vpc-ref.yaml

```diff
@@ -0,0 +1,14 @@
+ref:
+  as: ipi-conf-aws-local-zones-byo-vpc
```

Comment:
@mtulio Tasks in this step are currently done by `ipi-conf-aws-localzone` + `ipi-conf-aws-custom-vpc`,  I still think it's better to keep existing logic:
`ipi-conf-aws-custom-vpc` -> set subnet + zones
`ipi-conf-aws-localzone` -> set `compute[?name=edge]` + lz subnet,  and we can add more params in this step later